Tests/MustacheTests/TransformTests.swift

```diff
@@ -80,6 +80,61 @@ final class TransformTests: XCTestCase {
         """)
     }
 
+    /// Expected failure, but you're welcome to fix it if you find a way.
+    /// Use the recursive-transforms syntax instead, like the next test below.
+    func testNestedSequenceTransformDoesNotWork() throws {
```

Comment:
This is expected to fail as it is as Mustache is intended to work. I don't think we need this test.